src/components/ForgotPassword/ForgotPassword.jsx

```diff
@@ -1,7 +1,8 @@
 import React, { useState } from 'react';
 import { Auth } from 'aws-amplify';
-import { Link } from 'react-router-dom';
 import { Form, Button } from 'react-bootstrap';
+import styled from 'styled-components';
+import { ReactComponent as LockIcon } from './LockIcon.svg';
```

Comment:
I don't think you need to import this as a ReactComponent. It should just be an image.